hire/hireapp/migrations/0001_initial.py

```diff
@@ -0,0 +1,66 @@
+# Generated by Django 3.1.5 on 2021-01-30 00:17
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Company',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
```

Comment:
This file represents the database. This is what is used to update (migrate) the database.

You'll need to do a `./manage.py migrate` to apply these changes.